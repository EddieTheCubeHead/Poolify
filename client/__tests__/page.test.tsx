import { render, screen } from "@testing-library/react";
import React from "react";
import Page from "../app/page";

describe("Page", () => {
  it("renders homepage unchanged", () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
