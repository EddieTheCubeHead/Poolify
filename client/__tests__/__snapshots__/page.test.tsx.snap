// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders a About Stagnum section 1`] = `
<div>
  <main
    class="relative min-h-screen bg-white text-black flex flex-col w-full h-full"
  >
    <aside
      class="fixed flex flex-col bg-neutral-200 h-full px-12 py-16 w-96"
    >
      <ul
        class="flex flex-col gap-4"
      >
        <li>
          Home
        </li>
        <li>
          Search
        </li>
        <li>
          Create Pool
        </li>
        <li>
          Join Pool
        </li>
        <li>
          My Pool
        </li>
      </ul>
    </aside>
    <section
      class="relative flex flex-col h-full ml-96"
    >
      <button
        class="absolute right-12 h-24"
      >
        My Profile
      </button>
      <div
        class="fixed bg-black text-white bottom-0 left-0 w-full py-4 px-12 flex"
      >
        <div
          class="flex gap-4 w-96"
        >
          <a
            href="/about"
          >
            About Stagnum
          </a>
          <a
            class="text-[#1ED760]"
            href="https://github.com/EddieTheCubeHead/Stagnum/discussions"
            target="_blank"
          >
            Contact Us
          </a>
        </div>
        <p>
          Media Player
        </p>
      </div>
    </section>
  </main>
</div>
`;

exports[`Page renders homepage unchanged 1`] = `
<div>
  <main
    class="relative min-h-screen bg-white text-black flex flex-col w-full h-full"
  >
    <aside
      class="fixed flex flex-col bg-neutral-200 h-full px-12 py-16 w-96"
    >
      <ul
        class="flex flex-col gap-4"
      >
        <li>
          Home
        </li>
        <li>
          Search
        </li>
        <li>
          Create Pool
        </li>
        <li>
          Join Pool
        </li>
        <li>
          My Pool
        </li>
      </ul>
    </aside>
    <section
      class="relative flex flex-col h-full ml-96"
    >
      <button
        class="absolute right-12 h-24"
      >
        My Profile
      </button>
      <div
        class="fixed bg-black text-white bottom-0 left-0 w-full py-4 px-12 flex"
      >
        <div
          class="flex gap-4 w-96"
        >
          <a
            href="/about"
          >
            About Stagnum
          </a>
          <a
            class="text-[#1ED760]"
            href="https://github.com/EddieTheCubeHead/Stagnum/discussions"
            target="_blank"
          >
            Contact Us
          </a>
        </div>
        <p>
          Media Player
        </p>
      </div>
    </section>
  </main>
</div>
`;
